docs/streams/upgrade-guide.html

```diff
@@ -1104,6 +1104,12 @@ <h3 class="anchor-heading"><a id="streams_api_changes_0101" class="anchor-link">
         <li> JoinWindows has no default size anymore: JoinWindows.of("name").within(1000) changes to JoinWindows.of(1000) </li>
     </ul>
 
+    <p> New serde type <code>ListSerde</code>: </p>
+    <ul>
+        <li> added class <code>ListSerde</code> to (de)serialize <code>List</code>-based objects </li>
+        <li> introduced <code>ListSerializer</code> and <code>ListDeserializer</code> to power the new functionality </li>
+    </ul>
+
```

Comment:
Can you move this up to the top of this file, under the `Streams changes in 3.0` section? It's in reverse order, so the newest stuff goes at the top.